cmd/celestia-appd/cmd/root.go

```diff
@@ -82,10 +81,13 @@ func NewRootCmd() *cobra.Command {
 			}
 
 			// Override the default tendermint config for celestia-app
-			tmCfg := app.DefaultConsensusConfig()
-			customAppTemplate, customAppConfig := initAppConfig()
```

Comment:
[Question] There are other logics going on in the `initAppConfig()` to create the `customAppConfig` (which are deleted), doesn't it impact the content of the `customAppConfig`? do we maintain the same outcome? are these changes aligned with the amendment of the memppol TTL?

Reply:
We currently  don't make any modifications to the app config so `customAppConfig` is unused and misleading hence why I deleted it